arangosh/Import/ImportHelper.h

```diff
@@ -245,6 +249,14 @@ class ImportHelper {
 
   std::vector<std::string> getErrorMessages() { return _errorMessages; }
 
+  uint64_t getMaxUploadSize() {return(_maxUploadSize.load());};
```

Comment:
extra semicola after function body can be removed here and for following functions.

Reply:
done